### PR TITLE
Add failing test fixture for CountOnNullRector

### PR DESCRIPTION
**Pull request by melihovv**
# Failing Test for CountOnNullRector

Based on https://getrector.org/demo/1ec3365b-5712-6b86-83cb-0166e4fa413c

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Thanks for the test :+1: 

The code must be valid PHP. This one would fail: https://3v4l.org/coHt8

**Comment by melihovv** [comment]
@TomasVotruba changed example to this
```
<?php

namespace Rector\Tests\Php71\Rector\FuncCall\CountOnNullRector\Fixture;

final class Collection
{
    private array $items;
    
    public function __construct()
    {
        $this->items = [];
    }

    public function count(): int
    {
        return count($this->items);
    }
}
?>
```

File: rules-tests/Php71/Rector/FuncCall/CountOnNullRector/Fixture/collection.php.inc [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    {
+        $this->items = [];
+    }
+
+    public function count(): int
+    {
+        return count($this->items);
+    }
+}
+?>
```
Comment (samsonasik) [inline rules-tests/Php71/Rector/FuncCall/CountOnNullRector/Fixture/collection.php.inc]: this second part can be removed as equals:

```suggestion
```

**Comment by samsonasik** [comment]
@melihovv I cherry-picked your commit at PR https://github.com/rectorphp/rector-src/pull/1049

**Pull request closed without merging**